app/components/request-archive/component.js

```diff
@@ -0,0 +1,10 @@
+import Component from '@glimmer/component';
+
+export default class RequestArchiveComponent extends Component {
+  get allRequests() {
+    return this.args.model
+      .filter(({ status }) => status === 'closed')
+      .sortBy('createdAt')
+      .reverse();
+  }
```

Comment:
Name allRequests tells you that you can expect all requests but you get only closed ones. Consider renaming it to `allClosedRequests` or `closedRequests`